packages/starknet-snap/src/utils/starknetUtils.ts

```diff
@@ -37,14 +38,20 @@ import {
   validateAndParseAddress as _validateAndParseAddress,
   Signer,
   stark,
+  TransactionType as StarknetTransactionType,
 } from 'starknet';
 
-import type { RpcV4GetTransactionReceiptResponse } from '../types/snapApi';
+import type { EstimateFeeResponse } from '../rpcs/estimateFee';
```

Comment:
nit: 
the response should not refer to the rpc itself. some time it may lead the package confusion 
A reference B, B reference A

it may be better to delcare a type got the get estimate fee , although it may duplicate with the rpc